packages/knip/src/ConfigurationValidator.ts

```diff
@@ -77,6 +77,7 @@ export const pluginSchema = z.union([
 ]);
 
 const pluginsSchema = z.object({
+  wrangler: pluginSchema,
```

Comment:
Please move it down so it's ordered alphabetically

Reply:
Fixed in the latest commit